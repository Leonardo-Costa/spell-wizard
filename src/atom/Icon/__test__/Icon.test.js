import React from 'react';
import renderer from 'react-test-renderer';

import {Icon} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<Icon />).toJSON();
    expect(tree).toMatchSnapshot();
  });