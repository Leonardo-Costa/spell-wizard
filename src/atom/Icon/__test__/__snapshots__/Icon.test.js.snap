// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "height": undefined,
      "width": undefined,
    }
  }
>
  <Image
    resizeMethod="scale"
    style={
      Array [
        Object {
          "flex": 1,
          "width": undefined,
        },
        undefined,
      ]
    }
    tintColor={null}
  />
</View>
`;
